coding-standards/git.md

```
@@ -70,37 +167,104 @@ Pull Requests (PRs):
+
+8. Push your branch to GitHub:
+
+   ```shell
+   git push origin my-fix-branch
+   ```
+
+9. In GitHub, send a pull request to `main` branch.
+
+10. If the reviewer(s) suggests changes then:
+
+  - Make the required updates.
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)